scripts/azure-pipelines/linux-check.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+##############################################################################
+# Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and
+# other Serac Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+##############################################################################
+
+set -x
+
+function or_die () {
+    "$@"
+    local status=$?
+    if [[ $status != 0 ]] ; then
+        echo ERROR $status command: $@
+        exit $status
+    fi
+}
+
+or_die cd serac
+git submodule init 
+git submodule update 
+
+echo HOST_CONFIG
+echo $HOST_CONFIG
+
+echo "~~~~~~ RUNNING CMAKE ~~~~~~~~"
+or_die ./config-build.py -hc /home/serac/serac/host-configs/docker/${HOST_CONFIG}.cmake -DENABLE_GTEST_DEATH_TESTS=ON
```

Comment:
Why was the death test option given? make check doesn't run gtest.

Reply:
It was copy-pasted, will remove